odl/set/sets.py

```diff
@@ -217,6 +226,13 @@ def __contains__(self, other):
         characters, `False` otherwise."""
         return isinstance(other, basestring) and len(other) == self.length
 
+    def contains_all(self, array):
+        """Test if `array` is an array of strings with correct length."""
+        array = np.asarray(array)  # allow seqence type input
+        dtype_byte = np.dtype('S{}'.format(self.length))
+        dtype_uni = np.dtype('<U{}'.format(self.length))
+        return array.dtype in (dtype_byte, dtype_uni)
```

Comment:
Make sure to have tests for the use-cases you think about. We could optimize this later (perhaps a TODO)